src/Recommendations/RuleModelRepository.php

```diff
@@ -37,14 +39,14 @@ public function getLatestArticle()
     {
         // Research Advance, Research Article, Research Exchange, Short Report, Tools and Resources or Replication Study article
         $prepared = $this->db->prepare('
-          SELECT * FROM Rules 
-          WHERE Rules.type ="research-advance"
-          OR Rules.type ="research-article"
-          OR Rules.type ="research-exchange"
-          OR Rules.type ="short-report"
-          OR Rules.type ="tools-resources"
-          OR Rules.type ="replication-study" 
-          ORDER BY Rules.published DESC
+          SELECT * FROM '.$this->ruleTableName.' as Ru 
+          WHERE Ru.type ="research-advance"
```

Comment:
is this supposed to be used only in testing or also production? By the way, `WHERE RU.type IN (...)` might be faster due to index usage?